src/ceph-volume/ceph_volume/devices/__init__.py

```diff
@@ -1 +1 @@
-from . import lvm, simple # noqa
+from . import lvm, simple, rawbluestore # noqa
```

Comment:
I want to tie this to bluestore to avoid awkwardness when we have a different labeling scheme (and creation sequence etc) for seastore or whatever comes later.

Reply:
`raw` does make sense

Reply:
I think `raw` (the block mode) is independent of the osd store. So you can have `ceph-volume raw --bluestore` or at some point ``ceph-volume raw --seastar`.

Reply:
Yeah, it will just make the code harder to maintain; I expect seastar to be have pretty differently.  Is there any real difference is between ``ceph-volume raw --foo`` and ``ceph-volume rawfoo``?

Reply:
Its just a matter of consistency in the interface. Code wise this should make no difference, the first thing we do is dispatch commands based on the backing store. Ideally we'd have a structure like `c-v subcommand backingstore` as this would allow for proper argument grouping, but I think sticking with what is there is the least confusing way.

Reply:
@liewegas can you address this? (just like discussed yesterday), thanks!